.gitignore

```diff
@@ -5,3 +5,5 @@
 # font editor temporary files
 *(Autosaved).glyphs
 *.vfbak
+fonts/
```

Comment:
Per https://github.com/googlefonts/gf-docs/tree/master/Spec we should have a `/fonts/` folder with the VF and `/fonts/ttf` folder with statics